src/tests/mem/page_allocator.c

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @date 12.07.2018
+ * @author Filipp Chubukov
+ */
+
+#include <embox/test.h>
+#include <mem/page.h>
+#include <mem/heap.h>
+#include <mem/phymem.h>
+#include <util/binalign.h>
+
+EMBOX_TEST_SUITE("page_alloc_init test");
+
+TEST_CASE("Init page_allocator with one page") {
+	struct page_allocator *allocator;
+
+	extern char _reserve_end;
+	char *const phymem_alloc_start = (char *)
+		binalign_bound((uintptr_t) &_reserve_end, PAGE_SIZE());
+
+	const size_t mem_len = 4096;
```

Comment:
I guess mem_len should be equal to PAGE_SIZE(). If it is so, then it's better to write directly 'mem_len = PAGE_SIZE();'